pkgs/applications/misc/authenticator/default.nix

```diff
@@ -80,5 +78,8 @@ stdenv.mkDerivation rec {
     license = lib.licenses.gpl3Plus;
     maintainers = with lib.maintainers; [ austinbutler ];
     platforms = lib.platforms.linux;
+    # Fails to build on aarch64 with error
+    # "a label can only be part of a statement and a declaration is not a statement"
+    broken = stdenv.isLinux && stdenv.isAarch64;
```

Comment:
I was using Authenticator on Pinephone. It’s a shame that it’s broken now.

Reply:
Looks to me like an upstream bug. If you find a solution please create a PR.

```
FAILED: gsk/libgsk.a.p/gl_gskglrenderjob.c.o 
gcc -Igsk/libgsk.a.p -Igsk -I../gsk -I. -I.. -Igdk/x11 -I../gdk/x11 -Igdk -I../gdk -Igtk/css -I/nix/store/hayg4ar73n9s018lqkg490imx63ffqgy-glib-2.74.3-dev/include -I/nix/store/hayg4ar73n9s018lqkg490imx63ffqgy-glib-2.74.3-dev/include/glib-2.0 -I/nix/store/xkkys23phqgikyvj1bas5dq6v8k7k2zg-glib-2.74.3/lib/glib-2.0/include -I/nix/store/m30r0l3k448qvpp2gxmgwl8dxhc68vvh-graphene-1.10.8-dev/include/graphene-1.0 -I/nix/store/srw9riv8d3fjjaphk5cwbpr4x7hybfzc-graphene-1.10.8/lib/graphene-1.0/include -I/nix/store/kvqzbwkylr35787v3by8fi4lfrr0yval-harfbuzz-6.0.0-dev/include/harfbuzz -I/nix/store/3j4zj8vj6cxfhgd9xf8nkbwg33kqysgz-pango-1.50.12-dev/include/pango-1.0 -I/nix/store/wls937cdx58l405vsi5njyf331smsppj-cairo-1.16.0-dev/include/cairo -I/nix/store/ahmvpqld8dh07wxqmv8qxl5c0wpvnbl4-freetype-2.12.1-dev/include/freetype2 -I/nix/store/ahmvpqld8dh07wxqmv8qxl5c0wpvnbl4-freetype-2.12.1-dev/include -I/nix/store/xsn977sb7z5b6fhbp21myy3dqldyb0r2-gdk-pixbuf-2.42.10-dev/include/gdk-pixbuf-2.0 -I/nix/store/0dmg207m9dlgdjs475mznj5x41n3rcf2-fribidi-1.0.12/include/fribidi -I/nix/store/n93wnx35cb9siqrlr7dgv0hcanc811fk-libepoxy-1.5.10-dev/include -I/nix/store/4i7hpqghp60499c6l6vxp10sak700rba-fontconfig-2.14.0-dev/include -I/nix/store/hayg4ar73n9s018lqkg490imx63ffqgy-glib-2.74.3-dev/include/gio-unix-2.0 -I/nix/store/3afv6i7gp0db2s5mslc3d9c6kc18ph7g-libpng-apng-1.6.39-dev/include/libpng16 -I/nix/store/mwwvz7ia4r2nx4rc585jlmdlrcfz1rk7-libtiff-4.5.0-dev/include -I/nix/store/b5jdrbcvk2q40qvrjxqixq37vfgf8kz2-libjpeg-turbo-2.1.4-dev/include -I/nix/store/8fc3gfm0znvkfpcdfbdajnibnl0qgkyp-libxkbcommon-1.5.0-dev/include -I/nix/store/lr79z1h3ccn8fli7rmbr8i3lfy8j0lm5-wayland-1.21.0-dev/include -I/nix/store/743p97hgvrk6d3prw1q9h5h82ay8jy2q-xorgproto-2021.5/include -I/nix/store/24kqk7zpm0kpwnh17nc2qvzvzikpxmqj-libXrender-0.9.10-dev/include -I/nix/store/872j3mki34wq4701s5xmbfahsbz4j82m-libX11-1.8.3-dev/include -I/nix/store/4fbwjwn1846bbw41z24wk36dhqrprmif-libXi-1.8-dev/include -I/nix/store/arabqns9f6x8lrmbmq7ppr6mrdcxsrfv-libXext-1.3.4-dev/include -I/nix/store/65cppap8xgsvn1rawm3riz9xhi02b7yn-libXcursor-1.2.0-dev/include -I/nix/store/757xpjyd9985yaqd69clgq58ssdi4zda-libXdamage-1.1.5-dev/include -I/nix/store/4a09w9ql1hq36y6jfsvr0ly1l6lnkzid-libXfixes-6.0.0-dev/include -I/nix/store/9cac1adf9in3lh55kmm6pi7xsyw670ik-libXrandr-1.5.2-dev/include -I/nix/store/xdhjf9ns2mv54qaqmd9a6382m2dzfkzl-libXinerama-1.1.4-dev/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -std=gnu99 -DG_LOG_USE_STRUCTURED=1 -DGLIB_DISABLE_DEPRECATION_WARNINGS '-DGTK_VERSION="4.9.4"' -D_GNU_SOURCE -fPIC -DGTK_COMPILATION '-DG_LOG_DOMAIN="Gsk"' -DG_LOG_STRUCTURED=1 -fno-strict-aliasing -Wno-missing-include-dirs -Wduplicated-branches -Wduplicated-cond -Wformat=2 -Wformat-nonliteral -Wformat-security -Wignored-qualifiers -Wimplicit-function-declaration -Wlogical-op -Wmisleading-indentation -Wmissing-format-attribute -Wmissing-include-dirs -Wmissing-noreturn -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-prototypes -Wswitch-default -Wswitch-enum -Wundef -Wuninitialized -Wunused -Waddress -Warray-bounds -Wempty-body -Wimplicit -Wimplicit-fallthrough -Wimplicit-fallthrough=5 -Winit-self -Wint-to-pointer-cast -Wmain -Wmissing-braces -Wmissing-declarations -Wmissing-prototypes -Wnonnull -Wpointer-to-int-cast -Wredundant-decls -Wreturn-type -Wsequence-point -Wtrigraphs -Wvla -Wwrite-strings -Wcast-align -Wnull-dereference -fvisibility=hidden -MD -MQ gsk/libgsk.a.p/gl_gskglrenderjob.c.o -MF gsk/libgsk.a.p/gl_gskglrenderjob.c.o.d -o gsk/libgsk.a.p/gl_gskglrenderjob.c.o -c ../gsk/gl/gskglrenderjob.c
../gsk/gl/gskglrenderjob.c: In function 'gsk_gl_render_job_visit_texture_scale_node':
../gsk/gl/gskglrenderjob.c:3692:7: error: a label can only be part of a statement and a declaration is not a statement
 3692 |       float min_x = bounds->origin.x;
      |       ^~~~~
../gsk/gl/gskglrenderjob.c:3693:7: error: expected expression before 'float'
 3693 |       float min_y = bounds->origin.y;
      |       ^~~~~
../gsk/gl/gskglrenderjob.c:3695:21: error: 'min_y' undeclared (first use in this function); did you mean 'min_x'?
 3695 |       float max_y = min_y + bounds->size.height;
      |                     ^~~~~
      |                     min_x
../gsk/gl/gskglrenderjob.c:3695:21: note: each undeclared identifier is reported only once for each function it appears in
```

Reply:
Found this merged fix in GTK:
https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/5516

The fix is not available in gtk 4.9.4 but I tried gtk 4.10.0 and it works!

Reply:
Working on a PR at https://github.com/NixOS/nixpkgs/pull/222432.